cli/src/commands/mod.rs

```diff
@@ -545,6 +548,82 @@ struct NewArgs {
     insert_before: bool,
 }
 
+/// Move the current working copy commit to the next child revision in the
+/// repository.
+///
+///
+/// The command moves you to the next child in a linear fashion.
+///
+///
+/// D        D  @
+/// |        | /
+/// C  @ =>  C
+/// | /      |
+/// B        B
+///
+///
+/// If `--edit` is passed, it will move you directly to the child
+/// revision.
+///
+///
+/// D        D
+/// |        |
+/// C        C
+/// |        |
+/// B  @  => @
+/// | /      |
+/// A        A
```

Comment:
This diagram seems incorrect if this command is equivalent to `jj edit @+`. Using 1 space before, 1 space after the arrow:

```suggestion
/// D    D
/// |    |
/// C    C
/// |    |
/// B => @
/// |    |
/// @    A
```

Reply:
Both diagrams are actually correct, as `jj next --edit` is able to move in such a way. For clarity I used the simpler one.